python-flask/python-flask-example.py

```diff
@@ -0,0 +1,38 @@
+from flask import Flask, request, abort
+import base64
+import json
+import pprint
+import datetime
+app = Flask(__name__)
+
+@app.route('/', defaults={'path': ''}, methods=['POST'])
+@app.route('/<path:path>', methods=['POST'])
+def log_payload(path):
+    # Get request payload
+    if request.content_type != 'application/json':
+        # Unsupported Media Type
+        return 'Content type is not JSON', 415
+
+    # Ensure that payload is a dictionary
+    payload = request.json
+    if not isinstance(payload, dict):
+        # Payload not JSON, server error
+        return 'Payload is not JSON', 500
```

Comment:
No need to repeat ourselves

```suggestion
        return 'Payload is not JSON', 500
```

Reply:
They are two different errors (but maybe no clear enough), the first is if the `Content-type: application/json` is not set, if that is `true` it does not guarantee that the payload actually is valid JSON then this error would return.

Reply:
Fixed redundant comment